resources/sdk/purecloudjavascript/templates/configuration.mustache

```diff
@@ -0,0 +1,167 @@
+const ConfigParser = require('configparser');
+const os = require('os');
+const path = require('path');
+const fs = require('fs');
+
+import Logger from './logger.js';
+
+class Configuration {
+	/**
+	 * Singleton getter
+	 */
+	get instance() {
+		return Configuration.instance;
+	}
+
+	/**
+	 * Singleton setter
+	 */
+	set instance(value) {
+		Configuration.instance = value;
+	}
+
+	constructor() {
+		if (!Configuration.instance) {
+			Configuration.instance = this;
+		}
+
+		this.configPath = typeof window !== 'undefined' ? '' : path.join(os.homedir(), '.genesyscloudjavascript', 'config');
+		this.refresh_access_token = true;
+		this.refresh_token_wait_max = 10;
+		this.live_reload_config = true;
+		this.host;
+		this.environment;
+		this.basePath;
+		this.authUrl;
+		this.config;
+		this.logger = new Logger();
+		this.setEnvironment();
+		this.liveLoadConfig();
+	}
+
+	liveLoadConfig() {
```

Comment:
It doesn't seem to be live loading the config if using a custom path. It'll pick up the values initially but doesn't seem to be watching it

Reply:
Nice catch! After setting the custom config path, it needs to call liveLoad() again to reset the watcher. Fixed.